src/main.cpp

```diff
@@ -3944,15 +3945,16 @@ bool ContextualCheckBlock(
         CScript expect = CScript() << nHeight;
         if (block.vtx[0].vin[0].scriptSig.size() < expect.size() ||
             !std::equal(expect.begin(), expect.end(), block.vtx[0].vin[0].scriptSig.begin())) {
-            return state.DoS(100, error("%s: block height mismatch in coinbase", __func__), REJECT_INVALID, "bad-cb-height");
+            return state.DoS(100, error("%s: block height mismatch in coinbase", __func__),
+                             REJECT_INVALID, "bad-cb-height");
         }
     }
 
     // Coinbase transaction must include an output sending 20% of
-    // the block reward to a founders reward script, until the last founders
-    // reward block is reached, with exception of the genesis block.
-    // The last founders reward block is defined as the block just before the
-    // first subsidy halving block, which occurs at halving_interval + slow_start_shift
+    // the block subsidy to a Founders' Reward script, until the last Founders'
```

Comment:
`Founders' Reward` - Confusing for non English native speakers. 

Is it the accepted way to call it? i see it written different even inside the same ECC post for example: https://electriccoin.co/blog/electric-coin-company-q3-2019-transparency-report/

In the article, it refers to `Founders Reward` while in the tags at the bottom it is `#Founders' Reward`

Reply:
Yes, `Founders' Reward` is the correct term, as used in the original blog post: https://electriccoin.co/blog/funding/

Reply:
Thanks.